staging/src/k8s.io/pod-security-admission/test/run.go

```diff
@@ -122,6 +140,9 @@ func computeVersionsToTest(t *testing.T, checks []policy.Check) []api.Version {
 		// this can be incremented to the current version whenever is convenient.
 		// TODO: find a way to use api.LatestVersion() here
 		api.MajorMinorVersion(1, 23),
+		// Include the 1.25 release as pod restricted standard starts using Pod
+		// OS field
+		api.MajorMinorVersion(1, 25),
```

Comment:
Rather than _adding_ 1.25, I'd suggest changing 
```
api.MajorMinorVersion(1, 23),
```
to
```
api.MajorMinorVersion(1, newestMinorVersionToTest),
```

Because we declared new revisions of checks in 1.25, 1.25 will automatically be included, so we don't need a comment here about `the 1.25 release as pod restricted standard starts using Pod OS field`